src/github.com/matrix-org/dendrite/common/consumers.go

```diff
@@ -67,7 +67,9 @@ func (c *ContinualConsumer) Start() error {
 	}
 	for _, offset := range storedOffsets {
 		// We've already processed events from this partition so advance the offset to where we got to.
-		offsets[offset.Partition] = offset.Offset
+		// Offsets are provided with each message, so if we use the same offset on startup then we'll
```

Comment:
the logic looks right, but I find the comment confusing... isn't the point that the offset passed to `ConsumePartition` is inclusive?